docs/resources/rds_instance.md

```diff
@@ -200,6 +200,14 @@ The following arguments are supported:
   -> **NOTE:** async indicates the asynchronous replication mode. semisync indicates the semi-synchronous replication
   mode. sync indicates the synchronous replication mode.
 
+* `lower_case_table_names` - (Optional, String, ForceNew) Specifies the case-sensitive state of the database table name,
+  the default value is "1". Changing this parameter will create a new resource.
+    + 0: Table names are stored as fixed and table names are case-sensitive.
+    + 1: Table names will be stored in lower case and table names are not case-sensitive.
+
+  -> **NOTE:** When restoring an existing instance, the case setting of the table name of the target instance is
+  inconsistent with that of the original instance, which may cause the restoration to fail.
```

Comment:
we can delete this note as user can not restore the instance with terraform.

Reply:
ok, I have modified it